src/main/java/dev/esophose/playerparticles/styles/ParticleStyleArrows.java

```diff
@@ -64,6 +64,11 @@ protected ParticleStyleArrows() {
         return particles;
     }
 
+    @Override
+    public List<PParticle> getParticles(ParticlePair particle, Location location, Player player) {
```

Comment:
Remove all these after making the previous suggestion of using a default interface method.

Reply:
Didn't know about this method, thanks.